packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -158,9 +166,16 @@ type RecommendedComment = {
 }
 
 interface DialogueRecommendationRowProps {
-  rowProps: DialogueUserRowProps<boolean>; 
+  rowProps: {
+    targetUser: DialogueUserResult;
+    checkId: string | undefined;
+    hideInRecommendations: boolean;
+    userIsChecked: boolean;
+    userIsMatched: boolean;
+  }
```

Comment:
These should all be top-level props of the component.  (Wasn't this also in the last PR?)